src/tests/facility.test.js

```diff
@@ -0,0 +1,325 @@
+                });
+          })
+
+          it("should not delete facility", (done) => {
+            const data={
+              location:"test updated name ",
+              address:"test address ev updated 32'st",
+              images:"testImg.jpg",
+              roomNumber:"43",
+              roomDetails:"{sdlksldks:sdsds,sdsdsds:sdksjdskdj}"
+            }
+            chai
+              .request(app)
+              .delete(`/facilities/100`)
+              .set('authorization', travelAdminToken)
+              .end((err, response) => {
+                expect(response).to.have.status(404);
+                done();
+              });
+        })
+
+        it("should not delete facility", (done) => {
+          const data={
+            location:"test updated name ",
+            address:"test address ev updated 32'st",
+            images:"testImg.jpg",
+            roomNumber:"43",
+            roomDetails:"{sdlksldks:sdsds,sdsdsds:sdksjdskdj}"
+          }
+          chai
+            .request(app)
+            .delete(`/facilities/10-2`)
+            .set('authorization', travelAdminToken)
+            .end((err, response) => {
+              expect(response).to.have.status(404);
+              done();
+            });
+      })
+
+          it("should get all the facilities", (done) => {
+            chai
+              .request(app)
+              .get("/facilities")
+              .end((err, response) => {
+                expect(response).to.have.status(200);
+                expect(response.body).to.be.an("object");
+                done()
+                
+              });
+          });
+
+        })         
+    
+      });
+    })
+  })
+     
+  
+        
+    
+         
+   
+       
+      
+    
+        
+    
+
+  });
+
+  // /*
+  //  * GET one post using its (ID)
+  //  */
+  // describe("post facility", () => {
+    
+  //   it("should post facility by id", () => {
+  //     const _travelAdmin={
+  //       "email": "spacenomad@gmail.com",
+  //         "password": _travelAdminPwd
+  //     }
+     
+  //     chai
+  //       .request(app)
+  //       .get("/facility/" + id)
+  //       .end((err, response) => {
+  //         response.should.have.status(200);
+  //         response.body.expect.be.a("object");
+  //         response.body.expect.have.property("content");
+  //         response.body.should.have.property("id").eq(id);
+          
+  //       });
+  //   });
+
+  //   /*
+  //    * providing wrong id
+  //    */
+  //   // it("shouldn't get any post", () => {
+  //   //   const id2 = "5f4663e66450ec01f0adeed";
+  //   //   chai
+  //   //     .request(app)
+  //   //     .get("/facility/" + id2)
+  //   //     .end((err, response) => {
+  //   //       response.expect.have.status(404);
+  //   //       response.body.expect.be.a("object");
+  //   //       response.body.expect.have.property("error").eq("Post doesn't exist!");
+          
+  //   //     });
+  //   // });
+  // });  
```

Comment:
May you remove commented lines

Reply:
for me on my end, I don't have such coverage problem, also in your test directory I don't see facility.test.js which is supposed to contain test

![Screenshot from 2021-01-31 11-51-08](https://user-images.githubusercontent.com/59834399/106380406-acda8100-63ba-11eb-8fda-f829b43b985d.png)